### DISCUSSION TITLE
Possible to use zag/solid's useMachine with machines created with XState?

### DISCUSSION THREAD

**Discussion by bstro** [chakra-ui/zag#455] [Q&A] [answered]
Hi there, I'm curious if it is possible to directly use zag/solid's `useMachine` function with machines created with XState. It would be super handy, as I don't see any Solid/xstate integration at this point.

**Comment by anubra266** [comment, marked as answer]
It's not possible @bstro. Zag uses it's own state machine library under the hood.
